iam.tf

```diff
@@ -61,3 +68,39 @@ resource "aws_iam_role_policy_attachment" "CloudWatchAgentServerPolicy" {
   policy_arn = "arn:aws:iam::aws:policy/CloudWatchAgentServerPolicy"
   role       = aws_iam_role.worker_role.name
 }
+
+# Policy for VPC CNI with IPv6
+
```

Comment:
Should we add for documentation purposes where this policy comes from? (https://docs.aws.amazon.com/eks/latest/userguide/cni-iam-role.html#cni-iam-role-create-ipv6-policy or where you found it)

Reply:
Yes, I have a pending task to add quite a bit of documentation that I used to make all the changes. I just wanted first to make sure all the pieces fit together.